app/kafka/manager/utils/LogkafkaNewConfigs.scala

```diff
@@ -32,7 +32,9 @@ object LogkafkaNewConfigs {
     Kafka_0_10_2_1 -> logkafka82.LogConfig,
     Kafka_0_11_0_0 -> logkafka82.LogConfig,
     Kafka_0_11_0_2 -> logkafka82.LogConfig,
-    Kafka_1_0_0 -> logkafka82.LogConfig
+    Kafka_1_0_0 -> logkafka82.LogConfig,
```

Comment:
Please have a look at [this comment](https://github.com/yahoo/kafka-manager/pull/510#issuecomment-395349386).

Reply:
I will update